### PR TITLE
Travis ci

### PR DESCRIPTION
**Pull request by moshe010**
add Travis ci
and fix fix golangci-lint issues

### CODE REVIEW

File: sriovnet.go [hunk 1]
```diff
@@ -254,9 +256,11 @@ func SetVfPrivileged(handle *PfNetdevHandle, vf *VfObj, privileged bool) error {
 	/* do not check for error status as older kernels doesn't
 	 * have support for it.
 	 */
-	netlink.LinkSetVfTrust(handle.pfLinkHandle, vf.Index, trusted)
-	netlink.LinkSetVfSpoofchk(handle.pfLinkHandle, vf.Index, spoofChk)
-	return nil
+	err = netlink.LinkSetVfTrust(handle.pfLinkHandle, vf.Index, trusted)
```
Comment (paravmellanox) [inline sriovnet.go]: @moshe010 This breaks support on older kernels. Please rework this part to continue using existing way.

File: sriovnet_helper.go [hunk 2]
```diff
@@ -113,30 +113,3 @@ func GetVfPciDevList(pfNetdevName string) ([]string, error) {
 	}
 	return vfDirList, nil
 }
-
```
Comment (paravmellanox) [inline sriovnet_helper.go]: @moshe010 this change is unrelated to ci or travis.
Lets not mix multiple things under same subject.
Let it be a separate patch.

File: sriovnet.go (hunk 1 above)
Comment (moshe010) [reply sriovnet.go]: ok I revert the code to be like it was and put nolint to ignore the complains for now. I don't want this to block this PR.

File: sriovnet_helper.go (hunk 2 above)
Comment (lennybe) [reply sriovnet_helper.go]: recheck

**Comment by lennybe** [comment]
recheck

**Comment by lennybe** [comment]
@moshe010 you need to recommit this patch to trigger travis ci.
I've created a new PR with travis.yaml only [1] just to check it, you cab either merge mine and abondon yours, or make a new patch for this PR.
[1] https://github.com/Mellanox/sriovnet/pull/8

**Comment by moshe010** [comment]
@paravmellanox , Can we merge this? Travis CI is passing https://travis-ci.org/Mellanox/sriovnet/builds/531625616?utm_source=github_status&utm_medium=notification

**Pull request merged**